frontend/cypress/integration/components/mailingForm.spec.js

```diff
@@ -0,0 +1,70 @@
+describe('Mailing forms', () => {
+  it('checks labels and validation of a general form', () => {
+    // Visit engage page
+    cy.visit('/#/engage');
+    // Select general form tab
+    cy.get('[data-cy=general-form-tab]').click();
```

Comment:
Should this not be a check that it is automatically on this tab when a page is loaded? So remove the `.click` part.